crates/trigger/src/cli.rs

```diff
@@ -220,7 +219,19 @@ impl<T: Trigger<B::Factors>, B: RuntimeFactorsBuilder> FactorsTriggerCommand<T,
             follow_components,
             log_dir,
         };
-        let run_fut = builder.run(app, common_options, self.builder_args).await?;
+
+        // Create a component loader and enable AOT loading if the feature is enabled.
+        #[allow(unused_mut)]
+        let mut loader = ComponentLoaderImpl::new();
+        #[cfg(feature = "unsafe-aot-compilation")]
+        // Safety: This is safe because the feature is enabled.
+        unsafe {
+            loader.enable_loading_aot_compiled_components()
+        };
```

Comment:
I'm not sure that we want to enable this based just on the feature. At least, that isn't how it has worked previously.

Reply:
Whoops - I mistakenly remembered this being turned on in the CLI when compiled with the feature, but I suppose this function was only used by those consuming `spin-trigger` as a library. I'll remove this.